lib/teach-api.js

```diff
@@ -230,6 +230,12 @@ _.extend(TeachAPI.prototype, {
         ga.event({ category: 'Clubs', action: 'Deleted a Club' });
         callback(null);
       }.bind(this));
+  },
+  getClubsGuides: function(callback) {
+    callback = callback || function () {};
```

Comment:
The entire file does this though (literally copy/pasted from the other functions) so I think we should convert the entire file to ES6 in a separate patch